dod/src/test/java/com/revolut/rxdata/dod/DataObservableDelegateTest.kt

```diff
@@ -893,4 +893,28 @@ class DataObservableDelegateTest : BaseDataObservableDelegateTest() {
         verify(fromStorage, only()).invoke(eq(params))
         verify(toStorage, only()).invoke(eq(params), eq(domain))
     }
+
+    @Test
+    fun `WHEN previous LoadingStrategy is LazyReload AND memory cache is empty THEN observe LoadingStrategy Auto will reload`() {
```

Comment:
please add a test case that subsequent LazyReloads remain lazy, cause it seems to me that they're not

Reply:
fixed thx